blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -214,11 +214,15 @@ const Nav = (props) => {
 
 Nav.propTypes = {
   customFields: PropTypes.shape({
-    hierarchy: PropTypes.string,
+    hierarchy: PropTypes.string.tag({
+      label: 'Sections Menu hierarchy',
+      defaultValue: '',
+      group: 'Configure content',
+    }),
     signInOrder: PropTypes.number,
   }),
 };
 
-Nav.label = 'Header Nav Chain – Arc Block';
+Nav.label = 'Navigation - Arc Chain';
```

Comment:
Wouldn't this label conflict with https://github.com/WPMedia/fusion-news-theme-blocks/blob/canary/blocks/header-nav-block/features/navigation/default.jsx#L166 ?

Reply:
It looks like that feature has the label of `Navigation – Arc **Block**` whereas this one is `Navigation - Arc **Chain**` so I think it should be fine conflict-wise 🙂 Plus my understanding is that the `header-nav-block` will be deprecated in favor of the newer `header-nav-chain-block`.